schemas/zora/20210604.json

```diff
@@ -0,0 +1,96 @@
+{
+  "description": "This schema describes version 20210604 of the Zora Metadata Standard",
+  "title": "Zora20210604",
+  "$id": "https://zora.co/schemas/ZoraMetadata.json",
+  "$defs": {
+    "attribute": {
+      "type": "object",
+      "required": ["trait_type", "value"],
+      "properties": {
+        "trait_type": {
+          "type": "string",
+          "description": "The name of the trait"
+        },
+        "value": {
+          "type": ["string", "number", "boolean"],
+          "description": "The value of the trait"
+        },
+        "display_type": {
+          "type": "string",
```

Comment:
An enum feels weird here because the opensea spec is weird. Should be fine to leave this as-is.
"When omitted, defaults to the type passed into the value field"
if a number opensea shows a number, if a string opensea shows a string etc.